framework/src/modules/chain/chain.js

```diff
@@ -505,9 +509,19 @@ module.exports = class Chain {
 			this.channel.publish('chain:blocks:change', block);
 		});
 
+		this.transactionPool.on(EVENT_UNCONFIRMED_TRANSACTION, transaction => {
+			this.logger.debug(`Broadcasting transaction ${transaction.id}`);
+			this.transport.onUnconfirmedTransaction(transaction, true);
+		});
+
 		this.blocks.on(EVENT_NEW_BROADHASH, ({ broadhash, height }) => {
 			this.channel.invoke('app:updateApplicationState', { broadhash, height });
 		});
+
+		this.transactionPool.on(EVENT_MULTISIGNATURE_SIGNATURE, signature => {
+			this.logger.debug('Broadcasting signature ', signature);
```

Comment:
same as above

Reply:
> Maybe we could use Bunyan's metadata handling such as:
> `this.logger.debug({transactionId: 'theTrsId'}, 'Broadcasting transaction'}`
> This will print the logs in a parseable JSON-structured format

We could apply this here too

Reply:
Done